.mocharc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extension": ["ts"],
+  "spec": "**/**/*.test.ts",
+  "require": "test/babel-register.js"
+}
```

Comment:
[EOF](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

https://eslint.org/docs/rules/eol-last